packages/account-v2/src/modules/AddressFields/AddressFields.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useAuthorize, useSettings, useStatesList } from '@deriv/api';
+import FormDropDownField from '../../components/FormFields/FormDropDownField';
+import FormInputField from '../../components/FormFields/FormInputField';
+import { addressDetailValidations } from './validations';
+
+export const AddressFields = () => {
+    const { data: activeAccount } = useAuthorize();
+    const { data: settings } = useSettings();
+
+    const { landing_company_name: landingCompanyName, upgradeable_landing_companies: upgradableLandingCompanies } =
+        activeAccount;
+
+    const isSvg = landingCompanyName === 'svg' || !!upgradableLandingCompanies?.includes('svg');
```

Comment:
**NIT:** May be in the account-v2 package we can have some constants (`packages/account-v2/src/constants`), like `packages/cfd/src/Helpers/cfd-config.ts`

Reply:
Done.